parachains/integration-tests/collectives/0_xcm/0_init.yml

```diff
@@ -62,13 +62,26 @@ tests:
               pallet: xcmPallet
               call: send
               args: [
-                { v1: { 0, interior: { x1: { parachain: *cp_id }}}}, # destination
+                { v3: { 0, interior: { x1: { parachain: *cp_id }}}}, # destination
                 {
-                  v2: [ # message
+                  v3: [ # message
+                    {
+                      UnpaidExecution: {
+                        weightLimit: { 
+                          limited: {
+                            refTime: 2200000000, # 2_200_000_000
```

Comment:
same here, would nice to understand the logic behind weight values.

Reply:
so this just limit, use at most. 
1_000_000_000 per xcm instruction plus 200_000_000 for call 
not trying to make it precise or test anything related to weigths.